src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.js

```diff
@@ -606,20 +636,23 @@ const ImageAnnotationModalMap = ({
 }
 
 ImageAnnotationModalMap.propTypes = {
-  dataToReview: imageClassificationResponsePropType.isRequired,
-  setDataToReview: PropTypes.func.isRequired,
-  selectedAttributeId: PropTypes.string.isRequired,
-  hoveredAttributeId: PropTypes.string.isRequired,
   databaseSwitchboardInstance: PropTypes.object.isRequired,
-  setIsDataUpdatedSinceLastSave: PropTypes.func.isRequired,
+  dataToReview: imageClassificationResponsePropType.isRequired,
   getPointsGeojson: PropTypes.func.isRequired,
   getPointsLabelAnchorsGeoJson: PropTypes.func.isRequired,
   hasMapLoaded: PropTypes.bool.isRequired,
+  hoveredAttributeId: PropTypes.string.isRequired,
   imageScale: PropTypes.number.isRequired,
+  isTableShowing: PropTypes.bool.isRequired,
   map: PropTypes.object.isRequired,
+  patchesGeoJson: PropTypes.object.isRequired,
```

Comment:
This is required but it is initialized as undefined in ImageAnnotationModal. Will that cause issues?

Reply:
Nice catch! That was just me going too fast with copolot! Its not required. Will fix